plugins/woocommerce-admin/client/marketing/overview-multichannel/DiscoverTools/DiscoverTools.tsx

```diff
@@ -133,8 +133,75 @@ export const DiscoverTools = () => {
 
 					return (
 						<>
-							<CardDivider />
-							{ renderPluginCardBodies( subcategoryPlugins ) }
+							{ subcategoryPlugins.map( ( el ) => {
+								return (
+									<Fragment key={ el.product }>
+										<CardDivider />
+										<PluginCardBody
+											icon={
+												<img
+													src={ el.icon }
+													alt={ el.title }
+												/>
+											}
+											name={ el.title }
+											pills={ el.tags.map( ( tag ) => (
+												<Pill key={ tag.slug }>
+													{ tag.name }
+												</Pill>
+											) ) }
+											description={ el.description }
+											button={
+												el.direct_install ? (
```

Comment:
Should we add tests around this change?

Reply:
This is going to be also one of my feedback. I miss more tests for the feature.

Reply:
Good point! Added in [`c649ad0` (#35542)](https://github.com/woocommerce/woocommerce/pull/35542/commits/c649ad0f8df651ac2cd12ced0660e7366192e25e).